conans/client/migrations_settings.py

```diff
@@ -4029,3 +4029,5 @@
 """
 
 settings_1_54_0 = settings_1_53_0
+
+settings_1_55_0 = settings_1_54_0
```

Comment:
Interesting, this should have been done before, it seems it didn't and this is the first PR to 1.55.